models/migrations/migrations.go

```diff
@@ -176,6 +176,8 @@ var migrations = []Migration{
 	NewMigration("add is_fsck_enabled column for repos", addFsckEnabledToRepo),
 	// v61 -> v62
 	NewMigration("add size column for attachments", addSizeToAttachment),
+	// v62 -> v63
+	NewMigration("add last used passcode column for TOPT", addLastUsedPasscodeTOPT),
```

Comment:
Incorrect acronym, this should be `TOTP` (per: https://en.wikipedia.org/wiki/Time-based_One-time_Password_algorithm). There are other places in this PR that have the incorrect acronym.

Sorry for noticing this after my LG-TM, this should be an easy fix though.